x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -142,14 +153,18 @@ export async function installKibanaAssets(options: {
     managed: true,
   });
 
-  await makeManagedIndexPatternsGlobal(savedObjectsClient);
+  await makeManagedIndexPatternsGlobal(soClient);
 
   const installedAssets = await installKibanaSavedObjects({
     logger,
     savedObjectsImporter,
-    kibanaAssets: assetsToInstall,
+    kibanaAssets: assetsToInstall.filter((asset) => asset.type !== KibanaSavedObjectType.slo),
```

Comment:
I'm wondering if it makes sense to add `slo` to `KibanaSavedObjectType` and `KibanaAssetType` if it's not installed the same way as other kibana assets. Other types are filtered out in line 125.

SLOs could be installed in a separate function added [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/epm/packages/_install_package.ts), where each type of asset has its own function (kibana assets, ilm policies, etc.)

If we still think semantically SLOs belong in kibana assets, I think it makes sense to keep the installation in `installKibanaAssets`.

Reply:
+1 if they are installed differently it may be interesting to have his own assets type and persist it in the installation saved object as his own properties.

Having them mixed mean we will have to do some filtering in a lot of codepath deletion/upgrade and it will be easy to miss one.

Reply:
SLOs have saved objects but it also install few other resources like ingest pipelines and transorm, so can't be just bundled under traditional kibana saved objects.

Reply:
One do you think of introducing a new `slo_assets` key (by space if SLO are space aware) for the saved object installation? instead of using the kibana_assets one

Reply:
Also if we go that way it will be great to have a new step installSloAssets so that code is not mixed in the installKibanaAssets part

Reply:
yes SLOs are space aware.